lib/ifquery

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
+# Crudely emulate ifquery on systems that don't have it.
+
+import sys
+import glob
+
+def parse_config(interfaces_file, interface):
+    f = open(interfaces_file)
+    intf = ''
+    for line in f:
```

Comment:
[this may be me, not remembering Python correctly], but does this work ? 
i usually would do  `f.readlines()` or something of the sorts